spec/lib/wikitext_spec.rb

```diff
@@ -84,15 +84,46 @@
     let(:en_wiki) { Wiki.find_by(language: 'en', project: 'wikipedia') }
     let(:es_wiki) { create(:wiki, language: 'es', project: 'wikipedia') }
     let(:es_wiktionary) { create(:wiki, language: 'es', project: 'wiktionary') }
+    let(:en_wikinews) { create(:wiki, language: 'en', project: 'wikinews') }
+    let(:en_wikibooks) { create(:wiki, language: 'en', project: 'wikibooks') }
+    let(:pl_wikiquote) { create(:wiki, language: 'pl', project: 'wikiquote') }
+    let(:es_wikisource) { create(:wiki, language: 'es', project: 'wikisource') }
     let(:wikidata) { create(:wiki, project: 'wikidata') }
+    let(:de_wikiversity) { create(:wiki, language: 'de', project: 'wikiversity') }
+    let(:en_wikivoyage) { create(:wiki, language: 'en', project: 'wikivoyage') }
+    # Disabled due to Invalid Project:
+    # (`raise InvalidWikiError unless PROJECTS.include?(project)`) in app/models/wiki.rb
+    # let(:metawikipedia) { create(:wiki, language: 'en', project: 'metawikipedia') }
+    # let(:meta) { create(:wiki, language: 'en', project: 'meta') }
+    # let(:commons) { create(:wiki, language: 'en', project: 'commons') }
```

Comment:
Commons is a valid project, it's just done as `language: 'commons', project: 'wikimedia'`

Reply:
what do to with `Commons => c` shorthand then? should I do something like ` shorthands[language_prefix] || language_prefix`?